var/spack/repos/builtin/packages/py-ytopt-autotune/package.py

```diff
@@ -9,12 +9,16 @@
 class PyYtoptAutotune(PythonPackage):
     """Common interface for autotuning search space and method definition."""
 
-    maintainers = ['Kerilk']
+    maintainers = ['Kerilk', 'liuyangzhuan']
 
     homepage = "https://github.com/ytopt-team/autotune"
     url = "https://github.com/ytopt-team/autotune/archive/refs/tags/v1.1.0.tar.gz"
+    git      = "https://github.com/ytopt-team/autotune.git"
 
+    version('master', branch='master')
     version('1.1.0', sha256='5ee7fa6a1c83131c5ceba1537b25f00de84182e4d0e6ebd0fd6efa4e8aee1bc4')
 
+    patch('version.patch', when='@1.1.0')
```

Comment:
Has this patch been submitted upstream?

Reply:
Yes, I have a pull request here:
https://github.com/ytopt-team/autotune/pull/2